kvmagent/kvmagent/plugins/imagestore.py

```
@@ -162,30 +202,70 @@ def _parse_json_and_update_mode(volume_infos, json_data):
-                linux.rm_file_force(PFILE)
-                json_data = fd.read()
-                return  _parse_json_and_update_mode(volume_infos, json_data)
+        try:
+            with linux.ShowLibvirtErrorOnException(vm):
+                cmdstr = '%s cbtbak -domain %s -volumes %s -bitmap %s -portrange %s > %s' % \
+                         (self.ZSTORE_CLI_PATH, linux.shellquote(str(vm.uuid)), linux.shellquote(volumes),
+                          linux.shellquote(bitmapTimestamp), linux.shellquote(portRange), linux.shellquote(PFILE))
+                command_error = None
+                try:
+                    shell.call(cmdstr)
+                except Exception as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)